radiance_folder/model/README.md

```diff
@@ -0,0 +1,127 @@
+# Model folder
+
+Model folder includes all the geometry and material files that describes the Radiance
+model.
+
+```cmd
+└─model                :: model folder
+  ├───aperture         :: static apertures description
+  ├───aperture_group   :: apertures groups (AKA window groups)*
+  │   └───interior     :: interior aperture groups
+  ├───bsdf             :: in-model BSDF files and transmittance matrix files
+  ├───grid             :: sensor grids
+  ├───ies              :: electric lights description
+  ├───scene            :: static scene description
+  ├───scene_dynamic    :: dynamic scene description*
+  │   └───indoor       :: indoor dynamic scene description*
+  └───view             :: indoor and outdoor views
+```
+
+There is a `README.md` file in each sub-folder with more information about that
+sub-folder.
+
+# Folder structure in relation to Radiance studies
+
+This folder structure is designed to accommodate most of possible daylight workflows but
+it does not mean that all the files are necessary for all the simulations. For instance
+the separation of indoor aperture_group is only meaningful for 3 or 5-phase simulations
+where view matrix and daylight matrix calculation happens.
+
+Below is a quick overview of how the folder can accommodate the most commonly used
+Radiance workflows:
+
+
+## Ray-tracing with rtrace / rcontrib / rpict
+
+This is the most typical use case of Radiance. For static simulations all the geometries
+inside the `scene` folder will be compiled into a single `octree`.
+
+Since there are no view or daylight matrix calculation there is no difference between
+indoor and outdoor geometries. Also because there is no separate direct sunlight
+simulation the `.blk` files will not be used.
+
+```shell
+oconv ./model/scene/*.mat ./model/scene/*.rad \
+  ./model/aperture/*.mat ./model/aperture/*.rad > static_scene.oct
+```
+
+In `dynamic` models the `static apertures` should be separated from the rest of the the
+geometries. This is required for blacking-out the static aperture during the isolated
+contribution calculation.
+
+```shell
+oconv ./model/scene/*.mat ./model/scene/*.rad > static_scene_no_aperture.oct
+```
+
+For simulating the contribution of each isolated aperture the static_scene octree should
```

Comment:
`static_scene ` seems like a leftover from a previous iteration. It's just called `scene`, right?

Reply:
yes. changed it to `static scene`.